backend/src/api/auth/authSocial.ts

```diff
@@ -46,6 +46,26 @@ export default (app, routes) => {
       })(req, res)
     })
   }
+
+  if (GITHUB_CONFIG.clientId) {
```

Comment:
Let's also push these variables to crowd-kube for staging and prod

Reply:
I think the `clientId` is already present in the prod `env` since we use the same for integrations.